src/components/shared/VirtualList.js

```diff
@@ -244,22 +250,14 @@ type Geometry = {
   innerRectY: CssPixels,
 };
 
-class VirtualList extends React.PureComponent<VirtualListProps> {
-  _container: ?HTMLDivElement;
-  _inner: ?VirtualListInner;
+class VirtualList<Item> extends React.PureComponent<VirtualListProps<Item>> {
+  _container: {| current: HTMLDivElement | null |} = React.createRef();
+  _inner: {| current: VirtualListInner<Item> | null |} = React.createRef();
```

Comment:
I had to use the new refs to make Flow pass here; the old callback style wasn't working with the type parameter.

Note: I also tried this, thinking it would be better to read:
```js
   _container = React.createRef<HTMLDivElement>();
   _inner = React.createRef<VirtualListInner<Item>>();
```
but it wasn't passing either.
I also tried adding `: *` then, and then flow wasn't checking the usage properly anymore.

Reply:
Also strangely, I first forgot to actually call `createRef` but Flow didn't warn me :/ I think it should warn if a property that should be defined isn't at the end of the constructor :(

Reply:
I've noticed that Flow error before. The constructor mechanic isn't very robust. :-/

Reply:
so, I tried reproducing with a simpler example to at least report the error with `createRef` to flow, but everything I tried worked fine... So it should work most of the time, just it doesn't work well on our specific example here, and I don't know why :/